src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  for (const key in robot) {
+    const robotKey = robot[key];
+
+    if (robotKey in inversed) {
```

Comment:
it's better to use hasOwnProperty instead of `in`

Reply:
Done!